pages/contribution-flow.js

```diff
@@ -202,7 +202,16 @@ class NewContributionFlowPage extends React.Component {
       return <ErrorPage error={error} />;
     }
 
-    return <Page {...this.getPageMetadata()}>{this.renderPageContent()}</Page>;
+    return (
+      <Page
+        {...this.getPageMetadata()}
+        showFooter={false}
```

Comment:
This seems out of scope for this PR. Rebase issue?

Reply:
@Betree : As part of these changes we discussed that the footer should not be shown for contribution flow and the navbar should be minimized to show only the logo. So the idea here is that we have a contribution flow which is similar to [GoFundMe](https://www.gofundme.com/f/mtdrdc-texas-elementary-school-shooting-victims-fund/donate). This is why I included this on this PR as well. However if you think this is irrelevant we could extract this part to a separate PR or issue. 🤔

Reply:
It's difficult to understand how removing the footer in the contribution flow relates to "Add further refinements to navbar popups". So yes, I think we should move this change to a separate PR.